qa/suites/rados/basic/tasks/commands.yaml

```diff
@@ -0,0 +1,25 @@
+overrides:
+  ceph:
+    log-ignorelist:
+    - reached quota
+    - but it is still running
+    - overall HEALTH_
+    - \(POOL_FULL\)
+    - \(SMALLER_PGP_NUM\)
+    - \(CACHE_POOL_NO_HIT_SET\)
+    - \(CACHE_POOL_NEAR_FULL\)
+    - \(POOL_APP_NOT_ENABLED\)
+    - \(PG_AVAILABILITY\)
+    - \(PG_DEGRADED\)
```

Comment:
I think we can try changing this yaml to only have:

```
tasks:
  - cephfs_test_runner:
      modules:
        - tasks.tests.test_commands
```

suggest running this in teuthology:

`teuthology-suite ... -s rados:basic --filter commands`

to test.

Reply:
```"2023-09-18T10:17:05.436526+0000 mon.a (mon.0) 108 : cluster [WRN] Health check failed: 1 pool(s) do not have an application enabled (POOL_APP_NOT_ENABLED)" in cluster log```

this is what I wanted to avoid by copying the log ignore commands. See http://pulpito.front.sepia.ceph.com/leonidus-2023-09-18_09:48:49-rados:basic-wip-lusov-config-format-distro-default-smithi/

Reply:
Ah you see I totally forgot there was something here. I guess I still need you verdict